.github/blunderbuss.yml

```diff
@@ -184,4 +188,4 @@ assign_prs_by:
 - labels:
   - "api: bigquerydatatransfer"
   to:
-  - GoogleCloudPlatform/bigquery-data-connectors
\ No newline at end of file
+  - GoogleCloudPlatform/bigquery-data-connectors
```

Comment:
this team doesn't exist.

Reply:
I think editor whitespace is the thing that changed - the content wasn't modified.